.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = W391
+max-line-length = 80
+exclude = */tests/*
```

Comment:
Why are we ignoring tests?

Reply:
> Why are we ignoring tests?

@skapeyi  The exclude parameter is used to ignore file/dirs. In this case i made a mistake. But for this project i removed .flake8 , some we re not using it and hence we are not excluding tests.

Reply:
@nadralia remove the line `exclude = */tests/*` so that I can merge this.

Reply:
@nadralia never mind. I see you removed .flake8